pip-tools/dev-requirements.in

```diff
@@ -1,6 +1,6 @@
 -r ../requirements.txt
 black
-boto3-stubs[iam,cloudfront,elbv2]
+boto3-stubs[iam,cloudfront,elbv2]>1.34.33
```

Comment:
I'm curious: did we need to specify a version constraint here? Or would `upgrade-requirements` have upgraded it either way?

Reply:
thanks, I meant to remove that, I thought it was needed but meant to test without it and forgot. I don't think it's needed.